apache-cassandra-build-deps.pom

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>cassandra-parent</artifactId>
+    <groupId>org.apache.cassandra</groupId>
+    <version>4.1-alpha2-SNAPSHOT</version>
+    <relativePath>apache-cassandra-parent.pom</relativePath>
+  </parent>
+  <groupId>org.apache.cassandra</groupId>
+  <artifactId>cassandra-build-deps</artifactId>
+  <version>4.1-alpha2-SNAPSHOT</version>
+  <name>Maven Default Project</name>
+  <description>The Apache Cassandra Project develops a highly scalable second-generation distributed database, bringing together Dynamo's fully distributed design and Bigtable's ColumnFamily-based data model.</description>
+  <url>https://cassandra.apache.org/cassandra-build-deps</url>
+  <inceptionYear>2009</inceptionYear>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  <developers>
```

Comment:
How much of the duplicate metadata can be omitted, if this POM only exists to enumerate the build deps?

Reply:
Good point - already did a bunch of metadata cleanup (ant _write-poms used to put dependencyManagement in each child POM as well, for example), will continue to do so here.